src/spec/page-objects/settings-page.js

```diff
@@ -57,5 +58,13 @@ class SettingsPage {
       return val[0].click();
     });
   }
+  getDisabledDomain(){

```

Comment:
What does it means? `_domainsDisabled` is a collection, what is the result of `getText`?